src/components/landing/pages/intro/intro.tsx

```diff
@@ -1,90 +1,60 @@
 
     return (
-        <div id="home" className="section">
-            <div className="inner cover">
-                <h1 className="cover-heading">
-                    <FontAwesomeIcon icon="file-alt"/> CodiMD
-                </h1>
-                <p className="lead mb-5">
-                    <Trans i18nKey="coverSlogan"/>
-                </p>
-
-                {
-                    user.status === "forbidden" ?
-                        <Fragment>
-                            <div className="mb-5">
-                                <Link to="/login">
-                                    <Button
-                                        variant="success"
-                                        size="lg"
-                                        style={{minWidth: "200px"}}
-                                    >
-                                        <Trans i18nKey="signIn"/>
-                                    </Button>
-                                </Link>
-
-
-                                <span className="m-2">
-                                    <Trans i18nKey="or"/>
-                                </span>
-
-                                <Button
-                                    variant="primary"
-                                    size="lg"
-                                    href={`${url}/features`}
-                                    style={{minWidth: "200px"}}
-                                >
-                                    <Trans i18nKey="exploreFeatures"/>
-                                </Button>
-                            </div>
-
-                            <img alt="CodiMD Screenshot" src={screenshot} className="screenshot img-fluid mb-5"/>
-                        </Fragment>
-                    :
-                        null
-                }
-
-                <div className="lead row mb-5" style={{width: '90%', margin: '0 auto'}}>
-                    <div className="col-md-4 inner">
-                        <a href={`${url}/features#Share-Notes`} className="text-light">
-                            <FontAwesomeIcon icon="bolt" size="3x"/>
-                            <h5>
-                                <Trans i18nKey="featureCollaboration"/>
-                            </h5>
-                        </a>
-                    </div>
-                    <div className="col-md-4 inner">
-                        <a href={`${url}/features#MathJax`} className="text-light">
-                            <FontAwesomeIcon icon="chart-bar" size="3x"/>
-                            <h5>
-                                <Trans i18nKey="featureMathJax"/>
-                            </h5>
-                        </a>
+        <div>
+            <h1>
+                <FontAwesomeIcon icon="file-alt"/> CodiMD
+            </h1>
+            <p className="lead mb-5">
+                <Trans i18nKey="coverSlogan"/>
+            </p>
+
+            {
+                user.status === "forbidden" ?
```

Comment:
I would suggest to use `LoginStatus.forbidden` instead of the string comparison.